src/gui/tray/UserModel.h

```diff
@@ -7,6 +7,7 @@
 #include <QStringList>
 #include <QQuickImageProvider>
 
+#include <pushnotifications.h>
```

Comment:
Is this include needed?

Reply:
@FlexW Good catch. Since I've removed the enum, it's not anymore.